analysis/Scripts/NaiveYieldPlot.py

```diff
@@ -148,11 +213,15 @@ def main(argv):
 			
 			The parameters are
 				-h : help
-				-a : 2D histograms of each y vs. each pt (one for each year)
-				-b : ratio vs. pt (a line for each year)
-				-c : ratio vs. y (a line for each year)
-				-d : ratio vs. years 
-				-e : yield vs. year
+				-a : GaussCB 2D histograms of each y vs. each pt (one for each year)
+				-b : Bukin 2D histograms of each y vs. each pt (one for each year)
+				-c : GaussCB ratio vs. pt (a line for each year)
+				-d : Bukin ratio vs. pt (a line for each year)
+				-e : GaussCB ratio vs. y (a line for each year)
+				-f : Bukin ratio vs. y (a line for each year)
+				-g : ratio vs. years 
+
+				-i : yield vs. year
```

Comment:
@FelixRitter3110 that would be a very welcome check!

Reply:
I mean because you didn't add only new flags at the end (you changed e.g. -b to now be -c) and I wondered if there were places that this was called by other code - so their input code needs to be changed to reflect those new options.

Reply:
Oh in that case yes I checked that
thats why there is a gap between "-g" and "-i"; "-h" was already taken by the "help" option.

none of the other modes are used anywhere else

Reply:
I means there's no other parts of our code which runs NaivePlotYield.py -b for example ...?

Reply:
yes. checked and no conflicts